### PR TITLE
fix(favicons):logo in dark and light mode

### PR DESCRIPTION
**Pull request by doralchan**
**Before:**
Screenshot 2024-09-06 at 8 53 08 AM

**After:**
Screenshot 2024-09-06 at 12 19 20 PM

In order, Light Mode, Dark Mode and Dev Mode:
Screenshot 2024-09-06 at 12 17 55 PM

### CODE REVIEW

**Comment by evanpurkhiser** [comment]
Could you add some screenshots of what this looks like in the browser tab?

**Comment by evanpurkhiser** [comment]
I wonder if we should be providing various icon sizes that are each optimized for their own sizes?

File: static/app/utils/useColorscheme.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ function setFaviconTheme(theme: 'dark' | 'light'): void {
   }
 
   const path = faviconNode.href.split('/sentry/')[0];
-  const iconName = theme === 'dark' ? 'favicon-dark' : 'favicon';
+  const iconName = theme === 'dark' ? 'favicon-dev' : 'favicon';
```
Comment (evanpurkhiser) [inline static/app/utils/useColorscheme.tsx:24]: I'm not a fan of this. Now there's no easy way to differentiate between production sentry in dark mode and dev mode sentry.

Comment (doralchan) [reply static/app/utils/useColorscheme.tsx:24]: Oh that's a good point... ok, I'll get another color

**Review by scttcper — APPROVED** [review]

**Comment by scttcper** [comment]
no dash 😢

**Pull request merged**